### PR TITLE
One off code style reformat for all classes

### PR DESCRIPTION
**Pull request by cristianonicolai**

### CODE REVIEW

**Comment by cristianonicolai** [comment]
@nmirasch @bsremac code reformat only, targeting remaining classes from https://github.com/kiegroup/jbpm-wb/pull/782

File: jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/CaseActionsPresenter.java [hunk 1]
```diff
@@ -88,72 +89,109 @@ protected void loadCaseInstance(final CaseInstanceSummary cis) {
...
-                        addDynamicUserTaskAction(
-                                newActionView.getTaskName(),
-                                newActionView.getDescription(),
-                                newActionView.getActors(),
-                                newActionView.getGroups(),
-                                newActionView.getStageId()));
+            case DYNAMIC_USER_TASK: {
+                newActionView.show(caseActionType,
+                                   () ->
+                                           addDynamicUserTaskAction(
```
Comment (bsremac) [inline jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/CaseActionsPresenter.java]: Just one small remark on this one.
I think that starting method call in new line when called like this doesn't enhance readability.

File: jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/list/CaseInstanceListSearchViewImpl.java [hunk 2]
```diff
@@ -93,27 +93,35 @@ public HTMLElement getElement() {
...
     public void init() {
-        stream(CaseStatus.values()).forEach(s -> status.addOption(translationService.format(s.getLabel()), s.getName()));
+        stream(CaseStatus.values()).forEach(s -> status.addOption(translationService.format(s.getLabel()),
+                                                                  s.getName()));
         status.refresh();
 
-        stream(CaseInstanceSortBy.values()).collect(toMap(s -> s.name(), s -> translationService.format(s.name()))).entrySet().stream().sorted((e1, e2) -> e1.getValue().compareTo(e2.getValue())).forEach(s -> sortBy.addOption(s.getValue(), s.getKey()));
+        stream(CaseInstanceSortBy.values()).collect(toMap(s -> s.name(),
+                                                          s -> translationService.format(s.name()))).entrySet().stream().sorted((e1, e2) -> e1.getValue().compareTo(e2.getValue())).forEach(s -> sortBy.addOption(s.getValue(),
+                                                                                                                                                                                                                  s.getKey()));
```
Comment (bsremac) [inline jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/list/CaseInstanceListSearchViewImpl.java]: Not sure if anything can be done to this line to make it shorter and have that waterfall look.
Like this, that formatting at the end looks as if doesn't add any value in this case :)

File: jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/CaseActionsPresenter.java (hunk 1 above)
Comment (cristianonicolai) [inline jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/actions/CaseActionsPresenter.java]: Fixed

File: jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/list/CaseInstanceListSearchViewImpl.java (hunk 2 above)
Comment (cristianonicolai) [inline jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/list/CaseInstanceListSearchViewImpl.java]: I tried to improve it a bit, but there is a lot going on there, hopefully, it is a bit more readable.

Comment (bsremac) [inline jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-client/src/main/java/org/jbpm/workbench/cm/client/list/CaseInstanceListSearchViewImpl.java]: @cristianonicolai This looks way better now.

**Comment by bsremac** [comment]
+1

**Pull request merged**